src/archive.rs

```diff
@@ -46,9 +46,10 @@ impl Config {
         &self.uri
     }
 
+    #[allow(clippy::ref_option, clippy::clone_on_copy)]
```

Comment:
This `allow` attribute is no longer needed, since you've already fixed the clippy warnings.

Reply:
I'll remove it.
Thank you.